swiftide/src/integrations/qdrant/mod.rs

```diff
@@ -116,6 +153,26 @@ impl QdrantBuilder {
 
         Ok(Arc::new(client))
     }
+
+    /// Adds new [VectorConfig]
+    ///
+    /// When not configured Pipeline by default configures vector only for [EmbeddableType::Combined]
+    /// Default config is enough when [crate::indexing::Pipeline::with_embed_mode] is not set
+    /// or when the value is set to [crate::indexing::EmbedMode::SingleWithMetadata].
+    pub fn with_vector(mut self, vector: impl Into<VectorConfig>) -> QdrantBuilder {
+        if self.vectors.is_none() {
+            self = self.vectors(Default::default());
+        }
+        let vector = vector.into();
+        if let Some(vectors) = self.vectors.as_mut() {
+            vectors.insert(vector.embeddable_type.clone(), vector);
```

Comment:
Silently overrides vector.
The library has no logger, but it would deserve at least log msg.

Reply:
A disclaimer comment.
Resolving it to calm down PR merging rules.

Reply:
Done.